lib/src/main/java/adminlte/entity_list_table/communication/http/tables/columns/DateTimeColumn.java

```diff
@@ -63,23 +62,15 @@ private String getFormattedUtcDateTimeString(Object object) {
     private Instant getInstant(Object object) {
         var content = this.getObjectValue(object, this.fieldName);
 
-        if (content == null) {
-            return null;
-        }
+        if (content == null) { return null; }
 
-        if (content instanceof Instant) {
-            return (Instant) content;
-        }
+        if (content instanceof Instant) { return (Instant)content; }
 
-        if (content instanceof Date) {
-            return ((Date) content).toLocalDate().atStartOfDay().toInstant(ZoneOffset.UTC);
-        }
+        if (content instanceof Date) { return ((Date)content).toLocalDate().atStartOfDay().toInstant(ZoneOffset.UTC); }
 
         // assuming that LocalDateTime stores UTC value
-        if (content instanceof LocalDateTime) {
-            return ((LocalDateTime) content).toInstant(ZoneOffset.UTC);
-        }
+        if (content instanceof LocalDateTime) { return ((LocalDateTime)content).toInstant(ZoneOffset.UTC); }
```

Comment:
I think that auto formatting did something bad with code readability here.

Reply:
Yeah. Not critical though ?